lib/Field/FieldRepeatButton.js

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from 'react';
+import { bool, string } from 'prop-types';
+import cx from 'classnames';
+import { FieldNew } from 'lib';
+import { ButtonSecondary } from '../ButtonNew/ButtonSecondary/ButtonSecondary';
+import Icon from '../Icon';
+
+function ConnectorLine() {
+  return (
+    <span className="h-10 border-l border-0 border-solid border-neutral-90 " />
+  );
+}
+
+export function FieldRepeatButton({
```

Comment:
🚩I think this should really be two separate components, `FieldNew.FieldRepeatButton` and `FieldNew.FieldConnectorLine`.

Whenever we have a component that has an `if` statement in switching it between two different uses like this, it's usually a sign we would be better to factor the two use cases out to separate components.

This follows our single responsibility pattern, and we have found that although it is slightly more code, it is far easier to read than parsing a component with `if` statements switching things over.